driver/src/main/scala/api/collections/UpdateOps.scala

```diff
@@ -139,7 +139,7 @@ trait UpdateOps[P <: SerializationPack] extends UpdateCommand[P]
 
       BulkOps.bulkApply[UpdateElement, UpdateWriteResult](
         bulkProducer)(
-        { bulk => execute(firstUpdate, bulk.toSeq) },
```

Comment:
This break the safety to execute with at least on element. Not acceptable.

Reply:
Reverted. We change the original fix to minimise the diff.